locales/ru/filters.json

```
@@ -173,7 +173,7 @@
 	},
 	{
 		"filter.122.name": "Fanboy's Annoyances",
-		"filter.122.description": "Удаляет всплывающие окна на странице и другие неприятности. Включает списки cookie-файлов Fanboy's Social Blocking и EasyList."
+		"filter.122.description": "Удаляет всплывающие окна на странице и другие раздражающие элементы. Включает фильтры EasyList Cookie List и Fanboy's Social Blocking List."
```

`Включает в себя` сделал бы, чтобы не трактовалось как включение указанных фильтров в настройках.

```suggestion
		"filter.122.description": "Удаляет всплывающие окна на странице и другие раздражающие элементы. Включает в себя фильтры EasyList Cookie List и Fanboy's Social Blocking List."
```